### PR TITLE
[PDI-13375] - StepMeta clone doesn t clone over step attributesMap

### PR DESCRIPTION
**Pull request by akhayrutdinov**

### CODE REVIEW

File: engine/test-src/org/pentaho/di/trans/step/StepMetaTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+package org.pentaho.di.trans.step;
```
Comment (mattyb149) [inline engine/test-src/org/pentaho/di/trans/step/StepMetaTest.java]: Needs a copyright header

Comment (akhayrutdinov) [inline engine/test-src/org/pentaho/di/trans/step/StepMetaTest.java]: @mattyb149, have done it

**Comment by mattyb149** [comment]
I can't see in the provided Unit Test where your changes to replaceMeta() have been tested. Can you explain (or add the appropriate tests)?  Thanks!

**Comment by akhayrutdinov** [comment]
@mattyb149, oups, my fault. Fixed it by instantiating an attributes' map for a test instance. 

replaceMeta() is called from clone(), and the original ticket refers to problems during cloning routine, thus there is two steps in the test:
1. create a copy of a step by calling clone()
2. check whether its content is the same as the original has

**Pull request merged**